### PR TITLE
update Travis and Appveyor CI to SQ 5.6.1 and Scanner 2.6.1

### PR DESCRIPTION
**Pull request by guwirth**
update Travis and Appveyor CI to SQ 5.6.1 and Scanner 2.6.1
* change "...Cannot find the file..." from INFO to WARN
* change "...The report ... seems to be empty" from INFO to WARN
* ix Travis build, see travis-ci/travis-ci#5227 (comment)

### CODE REVIEW

**Comment by guwirth** [comment]
@jmecosta here also AppVeyor is failing. Maybe you can have a look....

**Comment by SonarOpenCommunityAdmin** [comment]
the only thing is see is this:

``
    But the analysis log contains a line matching
      """
      .*INFO.*Cannot find the file .* skipping violations
      """
      Traceback (most recent call last):
        File "c:\python27\lib\site-packagesehave\model.py", line 1456, in run
          match.run(runner.context)
        File "c:\python27\lib\site-packagesehave\model.py", line 1903, in run
          self.func(context, *args, **kwargs)
        File "integration-tests      eatures\steps	est_execution_statistics.py", line 267, in step_impl
          assert contains_line_matching(context.log, context.text)
      AssertionError
      
      Captured logging:
      INFO:requests.packages.urllib3.connectionpool:Starting new HTTP connection (1): localhost
      INFO:requests.packages.urllib3.connectionpool:Starting new HTTP connection (1): localhost
      INFO:requests.packages.urllib3.connectionpool:Starting new HTTP connection (1): localhost
    And the number of violations fed is 0
``

and now the scanner is given WARN instead of INFO

WARN: Cannot find the file 'C:\projects\sonar-cxx\integration-tests\testdata\cppcheck_project\component1.cc', skipping violations

Supose the feature needs to be updated, likely more of those should happen in Travis once is up

**Comment by guwirth** [comment]
@jmecosta thanks. Will change it from INFO to WARN and try it again.

File: integration-tests/features/environment.py [hunk 1]
```diff
@@ -296,14 +296,21 @@ def wait_for_sonar(timeout, criteria):
...
-        return urllib.urlopen(SONAR_URL).getcode() == 200
+        request = urllib.urlopen(SONAR_URL).getcode()
+        if request == 200:
+            sys.stdout.write(GREEN + " %3i " % (request) + RESET)
+            return True
+        else:
+            sys.stdout.write(RED + " %3i " % (request) + RESET)
+            return False
     except IOError:
+        sys.stdout.write(RED + " IOError " + RESET)
```
Comment (jmecosta) [inline integration-tests/features/environment.py]: can you change to this:
except Exception, e:
  sys.stdout.write(RED + " str(e.errno) " + RESET)
  sys.stdout.write(RED + " str(e) " + RESET)

so we can get error code about this.

**Pull request merged**